website/docs/d/google_storage_project_service_account.html.markdown

```diff
@@ -0,0 +1,38 @@
+---
+layout: "google"
+page_title: "Google: google_storage_project_service_account"
+sidebar_current: "docs-google-datasource-storage-project-service-account"
+description: |-
+  Get the email address of the project's Google Cloud Storage service account
+---
+
+# google\_storage\_project\_service\_account
+
+Use this data source to get the email address of the project's Google Cloud Storage service account.
+ For more information see 
+[API](https://cloud.google.com/storage/docs/json_api/v1/projects/serviceAccount).
+
+## Example Usage
+
+```hcl
+data "google_storage_project_service_account" "gcs_account" {}
+
+resource "google_pubsub_topic_iam_binding" "binding" {
+	topic       = "${google_pubsub_topic.topic.name}"
+	role        = "roles/pubsub.publisher"
+		  
+	members     = ["${data.google_storage_project_service_account.gcs_account.id}"]
+}
+```
+
+## Argument Reference
+
+There are no arguments available for this data source.
+
+## Attributes Reference
+
+The following attributes are exported:
+
+* `id` - The ID of the service account, which is its email_address
```

Comment:
any particular reason we need two attributes that are the same?

Reply:
All datasources have to have  ID set, and that's how they get stored in state file. I wanted to have email_address as an additional attribute, since it's consistent with what API returns and with what people would probably be looking for intuitevely. No real reason to have "email_address", other than consistency with API. What do you think? Leave as is or remove it ?